php-nginx/whitelist_functions.php

```diff
@@ -14,22 +14,33 @@
     $whitelist = array_filter(array_map('trim', explode(',', $whitelist)));
 }
 
-$suhosinBlacklist = ini_get('suhosin.executor.func.blacklist');
 $disable_functions = ini_get('disable_functions');
 
+// It's tricky, but we still don't have suhosin enabled and we can not fetch
+// the list from our suhosin config. Now it defines the list as a literal.
+// Note: We need to sync this list with our default suhosin config
+// (deb-package-builder/extensions/suhosin/debian/ext-suhosin.ini).
+// TODO: Find a better way
```

Comment:
Ok, I added back the `php-cli.ini` which is specific to the cli interface, so now we can simply run `whitelist_functions.sh` just after enabling suhosin. Thanks for the feedback.